src/cfn.py

```diff
@@ -63,7 +63,7 @@ def update(stackname, autostart="0", concurrency='serial'):
 
 @task
 @timeit
-def update_infrastructure(stackname):
+def update_infrastructure(stackname, skip=None):
```

Comment:
Could make it a whitelist?

Reply:
If it had a default of `ec2,sqs,s3` (to preserve safety of running `update_infrastructure` without arguments), but will make it slightly more complex to run `bldr update_infrastructure:journal--prod,components=[]`